src/room.cpp

```diff
@@ -186,6 +186,12 @@ void Room::BroadcastRoomInformation() {
 }
 
 MacAddress Room::GenerateMacAddress() {
-    // TODO(Subv): Generate a random mac address here.
-    return {};
+    MacAddress result_mac = NintendoOUI;
+    std::uniform_real_distribution<uint8_t> dis(0, 256); //Random byte between 0 and 0xFF
```

Comment:
wait.. it should also be 0, 255. You do it please - I've got other things to worry about currently.